docs/docsets/BoxSDK.docset/Contents/Resources/Documents/undocumented.json

```diff
@@ -254,829 +9,24 @@
     },
     {
       "file": "/Users/sgarlanka/Documents/SDKs/iOS/box-ios-sdk/Sources/Network/PagingIterator.swift",
-      "line": 13,
-      "symbol": "PagingParameter.offset(_:)",
-      "symbol_kind": "source.lang.swift.decl.enumelement",
-      "warning": "undocumented"
-    },
-    {
-      "file": "/Users/sgarlanka/Documents/SDKs/iOS/box-ios-sdk/Sources/Network/PagingIterator.swift",
-      "line": 14,
-      "symbol": "PagingParameter.marker(_:)",
-      "symbol_kind": "source.lang.swift.decl.enumelement",
-      "warning": "undocumented"
-    },
-    {
-      "file": "/Users/sgarlanka/Documents/SDKs/iOS/box-ios-sdk/Sources/Network/PagingIterator.swift",
-      "line": 15,
-      "symbol": "PagingParameter.streamPosition(_:)",
-      "symbol_kind": "source.lang.swift.decl.enumelement",
-      "warning": "undocumented"
-    },
-    {
-      "file": "/Users/sgarlanka/Documents/SDKs/iOS/box-ios-sdk/Sources/Network/PagingIterator.swift",
-      "line": 35,
+      "line": 37,
```

Comment:
There are a couple of stragglers in this file.  Do you wanna address them in this PR?

Reply:
Those have already been addressed. The undocumented.json file generated by jazzy was in the docs folder. Not sure why these didn't update.